src/components/HURUmap/Metrics/index.js

```diff
@@ -0,0 +1,50 @@
+import { Typography, LinearProgress } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+const Metrics = ({ percentage, description, color, summary }) => {
+  const classes = useStyles();
+
+  return (
+    <>
+      <div className={classes.root}>
+        <Typography variant="h3">{`${percentage}%`}</Typography>
```

Comment:
Metrics won't always be `percent values`.  `%`

Reply:
Oh..what are the other options so I can know what to show @KhadijaMahanga